VirtoCommerce.Platform.Web/Security/AuthenticationOptionsBasedPasswordCheckOptions.cs

```diff
@@ -0,0 +1,39 @@
+using VirtoCommerce.Platform.Core.Security;
+using VirtoCommerce.Platform.Data.Security;
+
+namespace VirtoCommerce.Platform.Web.Security
+{
+    /// <summary>
+    /// Implementation of <see cref="IPasswordCheckService"/> that gets password validation options from
+    /// <see cref="AuthenticationOptions"/>.
+    /// </summary>
+    public class AuthenticationOptionsBasedPasswordCheckOptions : IPasswordCheckOptions
```

Comment:
I guess better will be using already exists `AuthenticationOptions` type instead of this decorator